src/editors/mongoDBDocumentService.ts

```diff
@@ -147,7 +176,8 @@ export default class MongoDBDocumentService {
         return;
       }
 
-      return JSON.parse(EJSON.stringify(documents[0]));
+      const ejson = JSON.parse(EJSON.stringify(documents[0]));
+      return this._simplifyEJSON(ejson);
```

Comment:
What about a document that is like `{ nested: { property: UUID(...) } }`?

Reply:
This is one thing that I wanted to ask about actually. Can there be nested UUIDs, and can they be nested to any level?

Reply:
Yes, and yes :)